app/templates/crate/versions.hbs

```diff
@@ -12,7 +12,7 @@
 
 <div class="graph">
   <h4>Downloads per minor version in the last 90 days</h4>
-  {{versions-graph model=@model}}
+  {{versions-graph model=model}}
```

Comment:
This should be reverted, along with the linting configuration.

Reply:
Done